std/algorithm/package.d

```diff
@@ -54,6 +54,8 @@ $(TR $(TDNW Comparison)
         $(SUBREF comparison, min)
         $(SUBREF comparison, mismatch)
         $(SUBREF comparison, predSwitch)
+        $(SUBREF comparison, isSameLength)
+        $(SUBREF comparison, isPermutation)
```

Comment:
One last nit, because I saw Walter complain about this elsewhere. Put these in alphabetical order in the list (and the cheat sheet for the local module).